UPGRADE.md

```diff
@@ -59,6 +59,8 @@ There is a list of all the repositories maintained by monorepo, changes in log b
     - remove all `depends_on` and `links` from your docker-compose files because they are unnecessary
 - [#538 - phing targets: create-domains-data is now dependent on create-domains-db-functions](https://github.com/shopsys/shopsys/pull/538)
     - modify your `build.xml` and `build-dev.xml` according to this pull request
+- [#558 - Missing standards check in CI build process #558](https://github.com/shopsys/shopsys/pull/558)
+    - the Dockerfile for `php-fpm` has changed for CI stage build, update your `docker/php-fpm/Dockerfile`
```

Comment:
This should be just a note in the original upgrade note, imo

Reply:
I'm OK with merging this as-is, because the upgrade notes need fixing (eg. the upgrade note for #545 should be in `shopsys/project-base` section, not `shopsys/shopsys`). This should not block this PR.

I've updated our release-checklist and posted info about this on our Slack.